paddle/operators/sequence_erase_op.cu

```diff
@@ -23,13 +23,13 @@ using platform::PADDLE_CUDA_NUM_THREADS;
 using LoDTensor = framework::LoDTensor;
 
 template <typename T>
-__global__ void LabelErasedIdx(const T* in_dat, const int in_len,
-                               const T* tokens, const int tokens_len,
-                               int* num_erased) {
+__global__ void LabelErasedIdx(const T* in_dat, const int64_t in_len,
```

Comment:
Why in_len use `int64_t` while `tokens_len` is `size_t`?

Reply:
They have different data type.